static/elements/chromedash-metrics.js

```diff
@@ -74,7 +74,7 @@ class ChromedashMetrics extends LitElement {
   sort(e) {
     e.preventDefault();
 
-    const order = e.target.dataset.order;
+    const order = e.target.dataset.order || e.target.parentNode.dataset.order;
```

Comment:
`e.currentTarget` should cover both cases.

Reply:
Much better.  Thank you.